workspaces/server/src/infrastructure/repository/cloudflare/invites.ts

```diff
@@ -0,0 +1,60 @@
+import { eq, sql } from "drizzle-orm";
+import { type DrizzleD1Database, drizzle } from "drizzle-orm/d1";
+import * as schema from "../../../db/schema";
+import type {
+	IInvitesRepository,
+	InviteSchema,
+} from "../../../repository/invites";
+
+export class CloudflareInvitesRepository implements IInvitesRepository {
+	private client: DrizzleD1Database<typeof schema>;
+
+	constructor(db: D1Database) {
+		this.client = drizzle(db, { schema });
+	}
+
+	async createInvite(
+		expiresAt: Date | null,
+		remainingUse: number | null,
```

Comment:
```suggestion
		expiresAt?: Date,
		remainingUse?: number,
```
こうできたりしないかな (試してないのでわからんが)

Reply:
nullではなくundefinedになってしまいます

Reply:
いやそれは知ってるんだけど、 drizzle 側で undefined ←→ null 変換をよしなにやってくれた気がするので

Reply:
![image](https://github.com/user-attachments/assets/17844283-4e90-4e53-bf1a-26736b367faf)
response側で`Date | null`みたいに型が定義されていてうまく変換されてない...？
上手い書き方があったら教えてください...

Reply:
あ、ほんとだ、勘違いしてた
じゃあこのままでいきましょう～